src/datastation/scripts/open_access_archeodepot.py

```diff
@@ -0,0 +1,150 @@
+import argparse
+import json
+import logging
+import csv
+import re
+
+import psycopg
+
+from datastation.batch_processing import batch_process
+from datastation.config import init
+from datastation.dv_api import publish_dataset, get_dataset_metadata, change_access_request, replace_dataset_metadata, \
+    change_file_restrict
+
+
+def open_access_archeodepot(datasets_file, licenses_file, dag_raporten_file, dataverse_config, dry_run, delay):
```

Comment:
the `dag-raporten-file` is the list of files that should remain restricted? Maybe let the name reflect that.